src/refract/VisitorUtils.cc

```diff
@@ -0,0 +1,43 @@
+//
+//  refract/VisitorUtils.cc
+//  librefract
+//
+//  Created by Vilibald Wanča on 09/11/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+#include "Element.h"
+#include "Visitors.h"
+#include "VisitorUtils.h"
+namespace refract
+{
+
```

Comment:
If you are going to have `VisitorUtils.cc`, please move the other function definitions from `VisitorUtils.h` to the `.cc` file.